doc/sphinx/source/conformance.rst

```diff
@@ -113,146 +103,11 @@ Known issues related to CTS
   `<https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/2024>`_
   `<https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/2025>`_
 
-- Integer division by zero. OpenCL 1.2 specification requires that division by
-  zero on integers results in undefined values, instead of raising exceptions.
-  This requires pocl to install a handler of SIGFPE. The handler is per-process,
-  but it checks the thread ID, so that it only ignores the error for the CPU
-  driver threads, not the user program's threads. This might not work on every
-  system. The handler can be disabled completely by setting the env variable
-  POCL_SIGFPE_HANDLER to 0.
-  Note that this is currently only relevant for x86(-64) + Linux, on all other
-  systems this issue is not handled in any way (thus Pocl is likely
-  non-conformant there).
 
-- Many of ``native_`` and ``half_`` variants of kernel library functions are mapped
-  to the "full" variants.
-
-- clSetUserEventStatus() called with negative status. The Spec leaves the behaviour
-  in this case as "implementation defined", and this part of pocl is
-  only very lightly tested by the conformance tests. clSetUserEventStatus()
-  called with CL_COMPLETE works as expected, and is heavily used by
-  the conversions conformance test.
-
-Conformance tests results (precision of builtin math library functions)
+Conformance tests results
 -----------------------------------------------------------------------
 
-Note that it's impossible to test double precision on the entire range,
-therefore the results may vary.
-
-x86-64 CPU with AVX2+FMA, LLVM 4.0, tested on Nov 1, 2017
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+PoCL has submitted for OpenCL conformance in December 2024, and has been
```

Comment:
Why remove the title? It's important info related to which configuration and driver has been stamped? Also it would be good tag the exact commit which was approved in the repo.

Reply:
This is ancient information from 2017.  The valid current information about the setup is in the submission webpage, but i can copy some of it here.